concordium-std-derive/Cargo.toml

```diff
@@ -26,3 +26,4 @@ proc-macro = true
 [features]
 wasm-test = []
 build-schema = []
+concordium-quickcheck = []
```

Comment:
This feature must also be added to the CI. So we test with it.

Reply:
Done.